addons/mail/static/src/webclient/commands/mail_providers.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+// TODO LEGACY ENV
+import env from "web.env";
+
+const { Component } = owl;
+const { xml } = owl.tags;
+
+class DialogCommand extends Component {}
+DialogCommand.template = xml`
+    <t t-name="web.dialogCommand" owl="1">
+        <div class="o_command_default o_command_dialog">
```

Comment:
`o_command_default`? Isn't this specifically not a default one?

Also I don't understand what `o_command_dialog` is supposed to mean, and it doesn't look used anywhere.

Edit: ok I found it and I realized the rule is build dynamically with string manipulation, this is really not ideal for ease of understanding.